File: 3. Select Hello/Selective Hello.js
var stdIn = WScript.StdIn
var str = ""

stdOut.WriteLine("Name?");
	
while (!stdIn.AtEndOfStream)
{
	str = stdIn.ReadLine()
	if (str == "Bob") 
		{sayHello(str)}
	else if (str == "Alice")
		{sayHello(str)}
	else 
		{WScript.Echo ("Hello")}
	
	WScript.Quit();
}

function sayHello (name)
{
	WScript.Echo ("Hello " + name);
}

File: 3. Select Hello/Selective Hello.cs
using System;

class HelloCsharp
{
       static void Main()
       {
			string Name;
			
			Console.WriteLine("Name?");
			Name = Console.ReadLine ();
            if (Name == "Bob" || Name == "Alice")
				{ DisplayHello (Name);}
			else
				{ Console.WriteLine ("Hello"); }
       }
	   
	   static void DisplayHello(string Name)
	   {
			Console.WriteLine ("Hello " + Name); 
	   }
}